src/footer/userList/userList.js

```diff
@@ -35,8 +14,9 @@ export const Username = (props) => {
 
 export const ListComponentName = (props) => {
   return (
-    <div class="name">
-    <ul>{
-      props.usersList.map(user => <li key={user.id}>{user.fName} {user.lName} {user.age}</li>)
-    }</ul></div>
-  )};
+    <div className="name">
+      <Username fName='Dergaieva'  lName='Elena' age='30' />
+      <ul>{
+        props.usersList.map(user => <li key={user.id}>{user.fName} {user.lName} {user.age}</li>)
```

Comment:
```
props.usersList.map(user => <li key={user.id}>{user.fName} {user.lName} {user.age}</li>)
```
Inside < li>< /li> shoul be placed a Username component
there  was my [comment](https://github.com/Dergaeva/dergaeva.es-trainapp/pull/3#discussion_r154475697)
It is still not fixed